api/src/main/java/org/openmrs/module/formfilter/api/db/hibernate/HibernateFormFilterDAO.java

```diff
@@ -31,21 +32,21 @@
 	
 	protected final Log log = LogFactory.getLog(this.getClass());
 	
-	private SessionFactory sessionFactory;
+	private DbSessionFactory sessionFactory;
 	
 	/**
 	 * @param sessionFactory the sessionFactory to set
 	 */
-	public void setSessionFactory(SessionFactory sessionFactory) {
+	public void setSessionFactory(DbSessionFactory sessionFactory) {
 		this.sessionFactory = sessionFactory;
 	}
 	
 	/**
 	 * @return the sessionFactory
 	 */
-	public SessionFactory getSessionFactory() {
+	/*public SessionFactory getSessionFactory() {

```

Comment:
Did you see this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode

Reply:
yes sir , i saw that, it forbids commenting and encourages deprecating..